packages/cli/src/credentials/K8sCredentialManager.ts

```diff
@@ -237,8 +300,8 @@ class K8sCredentialManager extends AbstractCredentialManager {
      * @returns {string} the parsed string for the Kubernetes secret name
      */
     private getSecretName(account: string): string {
-        return `${account}-${this.kubeConfig.uid}`.replace(/_/g,'-').toLowerCase();
+        return `${account}-${this.kubeConfig.uid}`
+            .replace(/_/g, "-")
+            .toLowerCase();
     }
 }
-
-export = K8sCredentialManager;
```

Comment:
While I also dislike this style of import and generally support removing it, I think in this case we need to keep it because Imperative expects an unnamed export when loading this file as a credential manager override: https://github.com/zowe/imperative/blob/master/packages/security/src/CredentialManagerFactory.ts#L108

Reply:
We should probably address the ENOWORKSPACE error first before changing the export. Maybe I am wrong about the export and all the tests will pass once we change the npm version in the pipeline 🙂

Reply:
Currently pushed a new change, based on the results we can see whether we want to make changes to the export or not :)